tests/Application/.env.dist

```
@@ -21,3 +21,8 @@ DATABASE_URL=mysql://root@127.0.0.1/sylius_%kernel.environment%?serverVersion=5.
 # Delivery is disabled by default via "null://localhost"
 MAILER_URL=smtp://localhost
 ###< symfony/swiftmailer-bundle ###
+MESSENGER_TRANSPORT_DSN=sync://
```

```suggestion
###> symfony/messenger ###
MESSENGER_TRANSPORT_DSN=sync://
###< symfony/messenger ###
```

It doesn't matter a lot but if we used to mark which env var is from which Symfony component it can also be done here to be consistent.